### PR TITLE
Update food.go

### PR DESCRIPTION
**Pull request by pjindal**
more foods

### CODE REVIEW

**Review by prestonvanloon — APPROVED** [review]
I bet `!food` is the most popular feature @0xKiwi

File: food.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package main
 
 var foods = []string{
+	"Bahn Mi"
```
Comment (0xKiwi) [inline food.go:4]: YESSSS

File: food.go [hunk 2]
```diff
@@ -13,23 +14,35 @@ var foods = []string{
 	"Chicken Wings",
 	"Chili-Lime Salmon With Basmati Rice",
 	"Cumin Ground Lamb and Steamed Broccoli",
+	"Doner Kebab"
```
Comment (0xKiwi) [inline food.go:17]: omg yes

File: food.go [hunk 3]
```diff
@@ -13,23 +14,35 @@ var foods = []string{
...
 	"Fried Chicken",
+	"Fried Rice"
 	"Fries",
 	"Grilled Cheese",
+	"Gyro"
 	"Hash Browns",
 	"Hotpot",
+	"Kathi Rolls"
+	"Japanese Curry"
+	"Jerk Chicken with Carbbiean Fried Rice"
```
Comment (0xKiwi) [inline food.go:29]: YESSss pog

**Review by 0xKiwi — APPROVED** [review]
Amazing

**Comment by 0xKiwi** [comment]
Hey @pjindal this looks great! Would you mind signing the CLA for us? I know it's a very simple change but we require it haha

Thank you for this!

**Comment by pjindal** [comment]
All set 👍

**Pull request merged**